relaton/serializers/bibxml/reference.py

```diff
@@ -127,32 +132,26 @@ def create_reference(item: BibliographicItem) -> Element:
     else:
         ref.set('anchor', anchor)
 
-    # refcontent
-    refcontent = create_refcontent(item.extent)
-    if refcontent:
-        ref.set("refcontent", f"{refcontent}")
-
     return ref
 
 
-def create_refcontent(extent: Union[LocalityStack, Locality]) -> Union[None, str]:
-    refcontent = None
+def create_refcontent(extent: Union[LocalityStack, Locality, None]) -> Union[None, str]:
     if not extent:
         return None
-    if len(extent.locality) == 1:
-        extent: Locality = extent
-    else:
-        extent: LocalityStack = extent or []
     info = []
```

Comment:
I think `info` should be more descriptively named `parts`, because those are parts of the final string.